### ISSUE TITLE
cannot refresh configuration after manager update

### ISSUE THREAD

**Issue by linwalth** [osism/issues#75]
When I try to run `osism-manager configuration` after I have updated the manager to the new version, i get the following error (running with -vvv):

```
<$REDACTED_MGR_IP> ESTABLISH SSH CONNECTION FOR USER: dragon
<$REDACTED_MGR_IP> SSH: EXEC ssh -C -o ControlMaster=auto -o ControlPersist=60s -o StrictHostKeyChecking=no -o KbdInteractiveAuthentication=no -o PreferredAuthentications=gssapi-with-mic,gssapi-keyex,hostbased,publickey -o PasswordAuthentication=no -o 'User="dragon"' -o ConnectTimeout=10 -o ControlPath=/ansible/.ansible/cp/f59ac28b0b $REDACTED_MGR_IP '/bin/sh -c '"'"'echo ~dragon && sleep 0'"'"''
<$REDACTED_MGR_IP> (255, b'', b'dragon@$REDACTED_MGR_IP: Permission denied (publickey,password).\r\n')
fatal: [$REDACTED_MGR_HOSTNAME]: UNREACHABLE! => {
    "changed": false,
    "msg": "Failed to connect to the host via ssh: dragon@$REDACTED_MGR_IP: Permission denied (publickey,password).",
    "unreachable": true
}
```

**Comment by linwalth** [comment]
I have found additional information:

The extended error output from the failing command tells me that it is trying to access /ansible/.ssh/id_rsa

debug1: Next authentication method: publickey
debug1: Trying private key: /ansible/.ssh/id_rsa
debug3: no such identity: /ansible/.ssh/id_rsa: No such file or directory

Now the directory exists, but indeed it has no such key. it only has a known_hosts file.
The container does have the id_rsa.operator / id_rsa.configuration, but it resides in the /ansible/secrets directory. There is no link to the .ssh dir.

**Comment by linwalth** [comment]
Solved it. While, after an upgrade of the manager, one is told to move the vars of configuration.yml to the manager environment dir, one is not told to move the vars of the generic ansible.cfg accordingly, as the osism-manager run now takes on more responsibilities.
Since the old ansible.cfg does not know where to find the id_rsa.operator it will not be able to gather facts, hence halting the   osism-manager run at the first step.
I would recommend, therefore, to also alert users to the manager's newfound potential need for an extended ansible.cfg.

**Comment by berendt** [comment]
Fixed by osism/container-image-osism-ansible#198
